### ISSUE TITLE
Ruby 1.9.3 Support (public_suffix dependency)

### ISSUE THREAD

**Issue by bhelx** [bblimke/webmock#660]
I recently noticed that 1.9.3 builds in our travis matrix started breaking because the `public_suffix` gem, which is used by `addressable`, requires ruby >= 2.0. It seems like they dropped support for ruby 1.9.3 in version 1.5.0: https://github.com/weppos/publicsuffix-ruby/blob/master/CHANGELOG.md#release-150

And it seems like public_suffix was introduced to addressable in 2.5.0: https://github.com/sporkmonger/addressable/commit/f0326066e2888957cde5704360d840b0dfb722ac
which claims dropped support for ruby 1.9: https://github.com/sporkmonger/addressable/blob/master/CHANGELOG.md#addressable-250

So, is ruby 1.9.3 still supported as stated in the README? I don't mind relying on an older version but even when I went back to the last working version for me, I couldn't get it to build. Here is an attempt at building 1.7.10:

```
~ben ((v1.7.10)) ~/code/webmock: bundle
Fetching gem metadata from http://rubygems.org/...........
Fetching version metadata from http://rubygems.org/..
Fetching dependency metadata from http://rubygems.org/.
Resolving dependencies...
public_suffix-2.0.4 requires ruby version >= 2.0, which is incompatible with the
current version, ruby 1.9.3p551
```

Is it possible that the addressable dependency is too loosely defined? 

Thanks in advance

**Comment by bblimke** [comment]
1.9 is still supported. You can specify a compatible version of addressable in your Gemfile, next to webmock.

**Comment by bblimke** [comment]
unless you have a better idea?

**Comment by bhelx** [comment]
Thanks, I had not thought of this. I was thinking that setting the appropriate webmock version would also set a previous addressable version. I was able to fix by locking addressable to 2.4 in my gemspec:

```
s.add_development_dependency 'addressable',  '~> 2.4.0'
s.add_development_dependency 'webmock',  '~> 1.24.6'
```
